web/services/store/index.js

```diff
@@ -11,11 +11,9 @@ import localStorage from 'redux-persist/lib/storage';
 import hardSet from 'redux-persist/lib/stateReconciler/hardSet';
 
 import { firebaseConfig, rrfConfig } from '../../main.config';
+import Reducers from '../../reducers';
 
-const persistConfig = {
-  key: 'root',
-  storage: localStorage,
-};
+import { persistConfig } from './persist.config';
```

Comment:
any reason this is being moved out? it doesnt look like `persistConfig` is referenced elsewhere